### DISCUSSION TITLE
x64dbg installed in a Windows 10 Sandbox is not working ?

### DISCUSSION THREAD

**Discussion by Mer0winger** [x64dbg/x64dbg#3385] [General]
I tried to run it the latest download version within a Windows 10 Sandbox but it won´t work. It even started without outside Network connection ?
1) Why you need access to outside world ? 
2) It´s not trustable in this case i will not intercept my host security because of Windows Security is not trusting as well. 

So whats the Problem with using the Sandbox and why you need access to the internet ?

**Comment by mrexodia** [comment]
The latest version should not need internet access, this was a bug introduced because of the certificate checking. To completely disable this check you can use `x64dbg-unsigned.exe`.

**Reply by Mer0winger** [reply]
thanks its working
